Tests/Remora.Discord.Tests/Samples/Objects/WEBHOOK/2021-07-10.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Run this file through the scrubber first.

Reply:
Ah, right. I forgot about that. I asked Cisien first if he was cool with the sample being saved, but that makes more sense.

Reply:
Force-pushed to eliminate the unscrubbed file from history.